Filtering output plots for:
a_fischeri_ES114 a_kunk b_anth b_frag b_pseudo b_sub_168 b_theta b_xyla c_diff c_vibrioides d_vulg e_coli e_fae e_limo k_pneum m_smeg m_tuber n_gonorr p_cholor_aureo3084 p_fluor p_salmo s_coelicolor s_elon s_enter_typh s_epi s_meli s_sanguinis s_spcc6803 x_albi x_ory


Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_fischeri_ES114/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_fischeri_ES114/p2//*.csv’: No such file or directory
Filtering plots for a_fischeri_ES114
Finished filtering outputs for a_fischeri_ES114 into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_fischeri_ES114/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_kunk/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_kunk/p2//*.csv’: No such file or directory
Filtering plots for a_kunk
Clustering broke. Skipping.
Finished filtering outputs for a_kunk into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/a_kunk/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_anth/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_anth/p2//*.csv’: No such file or directory
Filtering plots for b_anth
Finished filtering outputs for b_anth into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_anth/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_frag/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_frag/p2//*.csv’: No such file or directory
Filtering plots for b_frag
Finished filtering outputs for b_frag into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_frag/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_pseudo/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_pseudo/p2//*.csv’: No such file or directory
Filtering plots for b_pseudo
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/scipy/stats/_morestats.py:3189: RuntimeWarning: invalid value encountered in scalar divide
  W = numer / denom
Finished filtering outputs for b_pseudo into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_pseudo/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_sub_168/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_sub_168/p2//*.csv’: No such file or directory
Filtering plots for b_sub_168
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Finished filtering outputs for b_sub_168 into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_sub_168/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_theta/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_theta/p2//*.csv’: No such file or directory
Filtering plots for b_theta
Finished filtering outputs for b_theta into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_theta/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_xyla/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_xyla/p2//*.csv’: No such file or directory
Filtering plots for b_xyla
Finished filtering outputs for b_xyla into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/b_xyla/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_diff/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_diff/p2//*.csv’: No such file or directory
Filtering plots for c_diff
Clustering broke. Skipping.
Clustering broke. Skipping.
Finished filtering outputs for c_diff into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_diff/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_vibrioides/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_vibrioides/p2//*.csv’: No such file or directory
Filtering plots for c_vibrioides
Finished filtering outputs for c_vibrioides into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/c_vibrioides/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/d_vulg/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/d_vulg/p2//*.csv’: No such file or directory
Filtering plots for d_vulg
Finished filtering outputs for d_vulg into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/d_vulg/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_coli/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_coli/p2//*.csv’: No such file or directory
Filtering plots for e_coli
Finished filtering outputs for e_coli into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_coli/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_fae/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_fae/p2//*.csv’: No such file or directory
Filtering plots for e_fae
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Clustering broke. Skipping.
Finished filtering outputs for e_fae into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_fae/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_limo/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_limo/p2//*.csv’: No such file or directory
Filtering plots for e_limo
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Clustering broke. Skipping.
Clustering broke. Skipping.
Clustering broke. Skipping.
Finished filtering outputs for e_limo into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/e_limo/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/k_pneum/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/k_pneum/p2//*.csv’: No such file or directory
Filtering plots for k_pneum
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Finished filtering outputs for k_pneum into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/k_pneum/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_smeg/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_smeg/p2//*.csv’: No such file or directory
Filtering plots for m_smeg
Clustering broke. Skipping.
Clustering broke. Skipping.
Clustering broke. Skipping.
Clustering broke. Skipping.
Finished filtering outputs for m_smeg into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_smeg/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_tuber/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_tuber/p2//*.csv’: No such file or directory
Filtering plots for m_tuber
Finished filtering outputs for m_tuber into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/m_tuber/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/n_gonorr/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/n_gonorr/p2//*.csv’: No such file or directory
Filtering plots for n_gonorr
Finished filtering outputs for n_gonorr into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/n_gonorr/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_cholor_aureo3084/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_cholor_aureo3084/p2//*.csv’: No such file or directory
Filtering plots for p_cholor_aureo3084
Finished filtering outputs for p_cholor_aureo3084 into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_cholor_aureo3084/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_fluor/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_fluor/p2//*.csv’: No such file or directory
Filtering plots for p_fluor
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-targeted", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-targeted')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/targeted/filter_BAM_plots.py", line 194, in exec_main
    main(
  File "/users/PDS0325/sachitk26/packages/tart/tart/targeted/filter_BAM_plots.py", line 332, in main
    peak_log, cluster_stats = Cluster(pd.DataFrame(log)).get()
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/cluster.py", line 25, in __init__
    self.stats_df = self.__compute_stats()
                    ^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/cluster.py", line 121, in __compute_stats
    varp = self.__var_test(peakset[self.statdim], exset[self.statdim])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/cluster.py", line 175, in __var_test
    return np.percentile(plist, 50)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py", line 4283, in percentile
    return _quantile_unchecked(
           ^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py", line 4555, in _quantile_unchecked
    return _ureduce(a,
           ^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py", line 3823, in _ureduce
    r = func(a, **kwargs)
        ^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py", line 4721, in _quantile_ureduce_func
    result = _quantile(arr,
             ^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py", line 4830, in _quantile
    slices_having_nans = np.isnan(arr[-1, ...])
                                  ~~~^^^^^^^^^
IndexError: index -1 is out of bounds for axis 0 with size 0
Finished filtering outputs for p_fluor into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_fluor/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_salmo/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_salmo/p2//*.csv’: No such file or directory
Filtering plots for p_salmo
Df empty for clustering. Returning unmodified argument.
Df empty for stats. Returning empty
/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_salmo/plots/picks.tar.gz seems empty. No tables exported.
Finished filtering outputs for p_salmo into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/p_salmo/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_coelicolor/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_coelicolor/p2//*.csv’: No such file or directory
Filtering plots for s_coelicolor
Finished filtering outputs for s_coelicolor into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_coelicolor/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_elon/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_elon/p2//*.csv’: No such file or directory
Filtering plots for s_elon
Clustering broke. Skipping.
Finished filtering outputs for s_elon into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_elon/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_enter_typh/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_enter_typh/p2//*.csv’: No such file or directory
Filtering plots for s_enter_typh
Finished filtering outputs for s_enter_typh into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_enter_typh/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_epi/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_epi/p2//*.csv’: No such file or directory
Filtering plots for s_epi
Finished filtering outputs for s_epi into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_epi/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_meli/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_meli/p2//*.csv’: No such file or directory
Filtering plots for s_meli
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Clustering broke. Skipping.
Finished filtering outputs for s_meli into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_meli/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_sanguinis/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_sanguinis/p2//*.csv’: No such file or directory
Filtering plots for s_sanguinis
Finished filtering outputs for s_sanguinis into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_sanguinis/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_spcc6803/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_spcc6803/p2//*.csv’: No such file or directory
Filtering plots for s_spcc6803
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:520: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/core/_methods.py:121: RuntimeWarning: invalid value encountered in divide
  ret = um.true_divide(
/users/PDS0325/sachitk26/packages/tart/tart/utils/activity_inference.py:338: RuntimeWarning: Degrees of freedom <= 0 for slice
  sd = np.cov(cov_nocand, rowvar=0)  # type: ignore
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: divide by zero encountered in divide
  c *= np.true_divide(1, fact)
/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/numpy/lib/function_base.py:2748: RuntimeWarning: invalid value encountered in multiply
  c *= np.true_divide(1, fact)
Finished filtering outputs for s_spcc6803 into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/s_spcc6803/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_albi/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_albi/p2//*.csv’: No such file or directory
Filtering plots for x_albi
Finished filtering outputs for x_albi into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_albi/p2/
----------------------------------------------------

Removing existing data
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_ory/p2//*/’: No such file or directory
rm: cannot remove ‘/users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_ory/p2//*.csv’: No such file or directory
Filtering plots for x_ory
Finished filtering outputs for x_ory into /users/PDS0325/sachitk26/packages/tart/validation/alignment/outputs/x_ory/p2/
----------------------------------------------------

Done
